src/content/ui-api/kubectl-gs/template-cluster.md

```diff
@@ -105,7 +116,11 @@ It supports the following flags:
   specify three distinct availability zones instead (AWS only). This can be done by separating AZ names with comma or using the flag
   three times with a single AZ name.
 
-### AWS specific
+### AWS specific (CAPI)
+
+ - `--name` must only contain alphanumeric characters, start with a letter, and be no longer than 5 characters in length
```

Comment:
Why 5? I remember we set a guideline to support a length of 10.

Reply:
For reference: https://github.com/giantswarm/roadmap/issues/1017

Reply:
As per our discussion `https://gigantic.slack.com/archives/C01F7T2MNRL/p1664271736738999` there is a validation currently happening in `kubectl gs` and a hidden flag to bypass the validation.  
This requires a new release of `Kubectl gs`  
So for now the documentation is correct in my opinion and we should address it once the changes are applied to `kubectl gs`  
I have opened an issue in Rainbow board: [ISSUE 1444](https://github.com/giantswarm/roadmap/issues/1444)

Reply:
ACK, this will be taken care of separately.